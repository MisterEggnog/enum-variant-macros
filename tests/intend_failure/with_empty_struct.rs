use enum_variant_macros::*;
#[derive(FromVariants)]
struct NotEnum;
fn main() {
}
