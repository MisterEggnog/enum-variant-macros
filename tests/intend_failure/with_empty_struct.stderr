error: Derive FromVariants for NotEnum failed. Must be an enum.
 --> tests/intend_failure/with_empty_struct.rs:3:1
  |
3 | struct NotEnum;
  | ^^^^^^
